### PR TITLE
strings and documentation fixes to src/modules

### PR DESCRIPTION
**Pull request by un1versal**
consistency, typos, capitalization, syntax, documentation indenting/formatting and other fixes

This is slightly past due (8 days) on my self assign limit date, after this we can push stuff to Transifex @ctrlaltca can you help with that please?

Edit --13/12/2015
Added need feedback lable in order to try get more eyes in case something slipped though, other than what Dessa pointed out, this needs to go in soon, Ive been running personal builds with this and nothing seems broken because of it.

### CODE REVIEW

**Comment by DarthGandalf** [comment]
> review with ?w=1

Please do whitespace changes and actual changes in separate commits

**Comment by un1versal** [comment]
thats just a nightmare with this amount of work, what good will that do?

have to split 90+ commits into singles and then hunt down those and separate them... thats just insane.

so this or each commit separately, please choose.

**Comment by DarthGandalf** [comment]
Ability to easily see the diff in any diff tool without relying on github.

Ability to see the diff is useful not only while PR is not merged yet, but also after, e.g. when searching in the git history.

**Comment by un1versal** [comment]
done as you asked, now be gentle. :D

**Comment by un1versal** [comment]
That last commit is a soup (literally) of random capitalization/consistency (mostly) fixes which was done after I split the whitspace/identing and because its mixed files cant merge with the first commit or second as it will error out in git, so to avoid more wasted time splitting all this into single commits and merging them into two groups the 3rd commit was done.

Im still going though my rather large list of notes and looking over dialogs to make sure I didnt miss something but its rather large amount so probably still something somewhere hiding from me.

**Comment by ctrlaltca** [comment]
> push stuff to Transifex @ctrlaltca can you help with that please?

Sure, I'll keep an eye on this ticket

**Comment by un1versal** [comment]
@ctrlaltca thx, if you could glance trough it or have any feedback on this as well would be nice in meawhile...

if we do a beta or something we need to have this included asap. Last I spoke to pragma he expressed a willingness to push a stable out around Christmas (quoting people have more time then) but after that conversation a few regressions were noted by @staticfox and some bugs should be fixed before then.

Then pragma also said that Christmas is also his busiest time, so who knows.

**Comment by staticfox** [comment]
As long as we don't make set-in-stone dates before we're ready to release, we should be fine.

**Comment by un1versal** [comment]
Well around Christmas is very vague as is, I would assume anything around up to new years also. Around Christmas though  is around sometime December.

My set date to have this last PR of these fix types done and up, was 10 days ago. at the speed of review here Im not expecting this merged for another week, but if it goes into two week Ill merge it anyway and deal with any fallout after.

File: data/config/serverdb.kvc [hunk 1]
```diff
@@ -407,9 +407,9 @@ NServers=1
 0_Hostname=irc.forestnet.org
 0_Description=Forestnet:%20Random%20server
 NServers=1
-[Freenode]
+[freenode]
```
Comment (Dessa) [inline data/config/serverdb.kvc:410]: im not sure if any stuff like favorites or network-based autoconnect is dependent on the capitalisation of the name...

File: src/modules/options/OptionsWidget_servers.cpp [hunk 2]
```diff
@@ -1317,34 +1317,34 @@ OptionsWidget_servers::OptionsWidget_servers(QWidget * parent)
 	m_pNewServerButton->setIcon(QIcon(*(g_pIconManager->getSmallIcon(KviIconManager::Server))));
 	m_pNewServerButton->setAutoRaise(true);
 	connect(m_pNewServerButton,SIGNAL(clicked()),this,SLOT(newServer()));
-	KviTalToolTip::add(m_pNewServerButton,__tr2qs_ctx("New Server","options"));
+	KviTalToolTip::add(m_pNewServerButton,__tr2qs_ctx("New nerver","options"));
```
Comment (Dessa) [inline src/modules/options/OptionsWidget_servers.cpp]: server

File: data/config/serverdb.kvc (hunk 1 above)
Comment (un1versal) [inline data/config/serverdb.kvc:410]: Seems to be case insensitive. Ive tested this and no ill side effects anywhere with auto-connect or reconnects on disconnect, unless Im missing something obvious.

@staticfox designed the favorites so hes more familiar with it.

**Comment by un1versal** [comment]
Any takers for a quick glance, checking if any obvious errors? If no takers in the morning will kick this in.

**Comment by un1versal** [comment]
merging, if anything was missed it will have to be solved after.

**Comment by un1versal** [comment]
@ctrlaltca Transifex time?

**Comment by ctrlaltca** [comment]
@un1versal strings extracted (#1795) and pushed to transifex. Translators should be able to do their job now.

**Pull request merged**